lib/routes/warp/blog.ts

```diff
@@ -0,0 +1,72 @@
+import { Route, DataItem, Data, ViewType } from '@/types';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import parser from '@/utils/rss-parser';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/blog',
+    categories: ['programming'],
+    example: '/warp/blog',
+    url: 'warp.dev',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['www.warp.dev'],
+            target: '/blog',
+        },
+    ],
+    name: 'Blog',
+    maintainers: ['cscnk52'],
+    handler,
+    description: `Provides a better reading experience (full articles) over the official ones.`,
+    view: ViewType.Notifications,
+};
+
+async function handler() {
+    const feed = await parser.parseURL('https://www.warp.dev/blog/rss.xml');
+
+    const items = await Promise.all(
+        feed.items.map((item) =>
+            cache.tryGet(item.link as string, async () => {
+                const data = await ofetch(item.link as string);
+                const $ = load(data);
+
+                const main = $('main');
+
+                // clean HTML
+                main.find('style').remove();
+                main.find('[style]').removeAttr('style');
+                main.find('[class]').removeAttr('class');
+                main.find('[id]').removeAttr('id');
+                main.find('script').remove();
```

Comment:
Please remove the title, time and the back button at the beginning of the article.